src/trace_redaction/BUILD.gn

```diff
@@ -32,6 +32,8 @@ source_set("trace_redaction") {
     "add_synth_threads_to_process_trees.h",
     "broadphase_packet_filter.cc",
     "broadphase_packet_filter.h",
+    "collect_clocks.cc",
```

Comment:
As previously mentioned, the general practice for the refactor code is to implement each primitive in its own CL/PR. It makes it easier to track when each primitive was added and makes it easier to document the dependency between them.

Since primitives must be implemented to be statically independent (functionally they connect through the context) this should be the simplest change compared to the trace processor changes.

Changes to trace redaction and trace processor should never happen in the same CL/PR as they are considered independent "products" - they only share the protobuf.